Gems/Atom/Tools/MaterialEditor/Code/Source/MaterialEditorApplication.h

```diff
@@ -22,6 +22,7 @@
 #include <AzToolsFramework/API/EditorPythonConsoleBus.h>
 #include <AzToolsFramework/Logger/TraceLogger.h>
 #include <AzQtComponents/Application/AzQtApplication.h>
+#include <AtomToolsFramework/Application/AtomToolsApplication.h>
```

Comment:
You may clean up some includes here since AtomToolsApplication.h already includes most of them